<launch>

  <node pkg="m2g_racing" name="tune_image_brightness_left" type="tune_image_brightness" output="screen">
    <remap from="/image_raw" to="airsim_node/drone_1/front_left/Scene"/>
    <remap from="/output_video" to="output_video/left/Scene"/>
  </node>

  <node pkg="m2g_racing" name="tune_image_brightness_right" type="tune_image_brightness" output="screen">
    <remap from="/image_raw" to="airsim_node/drone_1/front_right/Scene"/>
    <remap from="/output_video" to="output_video/right/Scene"/>
  </node>

</launch>
